src/components/show-edit-pod/show-edit-pod.stories.mdx

```diff
@@ -0,0 +1,183 @@
+- Viewing content that’s grouped together visually? [Try Pod](/?path=/docs/pod--basic "Pod").
+
+## Quick Start
+
+```jsx
+import ShowEditPod from "carbon-react/lib/components/show-edit-pod";
+
+const MyComponent = () => (
+  return (
+    <ShowEditPod title="Title">
+      <Content title="Content">
+        Content
+      </Content>
+    </ShowEditPod>
+  )
+);
+```
+
+### Basic
+
+<Preview>
+  <Story name="Basic">
+    <ShowEditPod 
+      title="Title"
+      onEdit={() => {}}
+      onDelete={() => {}}
+      saveText="Save"
+      deleteText="Delete"
+      cancelText="Cancel"
+      editFields={
+        <Textbox
+          label="Field"
+          labelInline
+          labelAlign='right'
+        />
+      }
+    >
+      <Content title="Content title">
+        Content
+      </Content>
+    </ShowEditPod>
+  </Story>
+</Preview>
+
+### Buttons alignment
+
+<Preview>
+  <Story name="Buttons alignment">
+    {() => ['left', 'right'].map(buttonAlign =>
+      <>
+        <ShowEditPod 
+          title={buttonAlign}
+          onEdit={() => {}}
+          onDelete={() => {}}
+          saveText="Save"
+          deleteText="Delete"
+          cancelText="Cancel"
+          buttonAlign={buttonAlign}
+          editFields={
+            <Textbox
+              label="Field"
+              labelInline
+              labelAlign='right'
+            />
+          }
+        >
+          <Content title="Content title">
+            Content
+          </Content>
+        </ShowEditPod>
+        <div style={{ marginBottom: 24 }}/>
+      </>
+    )}
+  </Story>
+</Preview>
+
+### With borders
+
+<Preview>
+  <Story name="With borders">
+    <ShowEditPod 
+      title="With borders"
+      onEdit={() => {}}
+      onDelete={() => {}}
+      saveText="Save"
+      deleteText="Delete"
+      cancelText="Cancel"
+      border
+      editFields={
+        <Textbox
+          label="Field"
+          labelInline
+          labelAlign='right'
+        />
+      }
+    >
+      <Content title="Content title">
+        Content
+      </Content>
+    </ShowEditPod>
+  </Story>
+</Preview>
+
+### Different variants
+
+<Preview>
+  <Story name="Different variants">
+    {() => [
+      'primary',
+      'secondary',
+      'tertiary',
+      'tile',
+      'transparent'
+    ].map(variant =>
+      <>
+        <ShowEditPod 
+          title={variant}
+          onEdit={() => {}}
+          onDelete={() => {}}
+          saveText="Save"
+          deleteText="Delete"
+          cancelText="Cancel"
+          variant={variant}
+          editFields={
+            <Textbox
+              label="Field"
+              labelInline
+              labelAlign='right'
+            />
+          }
+        >
+          <Content title="Content title">
+            Content
+          </Content>
+        </ShowEditPod>
+        <div style={{ marginBottom: 24 }}/>
+      </>
+    )}
+  </Story>
+</Preview>
+
```

Comment:
Could we add demonstration of how to use a custom transition between view states?